.github/workflows/release-opamp-client-node.yml

```diff
@@ -0,0 +1,57 @@
+# Release a tagged version of the '@elastic/opamp-client-node' package.
+name: release-opamp-client-node
+
+on:
+  push:
+    tags:
+      - opamp-client-node-v*.*.*
+
+# 'id-token' perm needed for npm publishing with provenance (see
+# https://docs.npmjs.com/generating-provenance-statements#example-github-actions-workflow)
+permissions:
+  contents: write
+  pull-requests: read
+  id-token: write
+
+jobs:
+  release:
+    runs-on: ubuntu-24.04
+    env:
```

Comment:
same comment as earlier let's have all the custom permissions here, apart from `permissions.contents: read` which ought to live at the top level.